src/containers/EndingContainer.jsx

```diff
@@ -185,8 +186,8 @@ class LastPage extends React.PureComponent {
 export default LastPage;
 
 export const query = graphql`
-  query($language: String!) {
-    locales: allLocale(filter: { language: { eq: $language } }) {
+  query {
```

Comment:
the `$language` filter is removed from all graphql queries. If it is used then only the current locale will be loaded and fallback keys will fail. There is perhaps a way we could alter this to load both the current locale plus the default locale so it does not query every single language available.